start-daemon.sh

```diff
@@ -1,2 +1,13 @@
 #!/bin/bash
+
+# inititiate Flutter app
+if ! command -v flutter &> /dev/null
+then
+    echo "Error: Flutter not installed" >&2
+else
+    echo -e "ADDRESS=<hostname>\nPORT=8080" > .env
+    flutter build web
+    cp build/web ddv
+fi
+
```

Comment:
I'm not totally sure that this script is correct. `flutter build web` (as well as the following line) needs to be run from the `rubintv_visualization` directory. So you'll have to `cd` to that directory, run the install commands, and `cd` back into the main path (which also means chaning `cp build/web ddv` to `cp build/web ../ddv`.